draft-ietf-rats-architecture.md

```diff
@@ -961,23 +1001,27 @@ Carsten Bormann provided many of the motivational building blocks with respect t
 
 The table below defines a number of relevant events, with an ID that
 is used in subsequent diagrams.  The times of said events might be
-defined in terms of an absolute clock time such as Coordinated Universal Time,
-or might be defined relative to some other timestamp or timeticks counter.
+defined in terms of an absolute clock time such as Coordinated Universal Time, 
+or might be defined relative to some other timestamp or timeticks counter with different timescales.
 
 | ID | Event                       | Explanation of event
 |----|-----------------------------|-----------------------
-| VG | Value generation            | A value to appear in a claim was created
-| NS | Nonce sent                  | A random number not predictable to an Attester is sent
-| NR | Nonce relayed               | The nonce is relayed to an Attester by enother entity
-| EG | Evidence generation         | An Attester collects claims and generates Evidence
-| ER | Evidence relayed            | A Relying Party relays Evidence to a Verifier
-| RG | Result generation           | A Verifier appraises Evidence and generates an Attestation Result
-| RR | Result relayed              | A Relying Party relays an Attestation Result to a Relying Party
-| RA | Result appraised            | The Relying Party appraises Attestation Results
+| VG | Value generation            | A value to appear in a Claim was created.
+| AA | Attester awareness          | An Attesting Environment starts to be aware of a new/changed Claim value.
+| CC | Claim Collection            | An Attesting Environment collects a new/changed Claim value to appear in Evidence.
+| HD | Handle distribution         | A centrally generated identifier for time-bound recentness across a domain of devices is successfully received by Attesters (and Verifiers).
```

Comment:
I think these new-added events had been questioned before, that the examples below don't use them.

Reply:
This chunk of text is gone.